app/Resources/TwigBundle/views/Exception/error503.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends '::_base.html.twig' %}
+
+
+{% block body %}
+    <h1>Oops! An Error Occurred</h1>
+    <h2>The server returned a "{{ status_code }} {{ status_text }}".</h2>
+
+    <div>
+        {{ exception.message }}
```

Comment:
Well, I never write exceptions thinking that a final user could read it but if you're sure that this is what we want go ahead :)  I'll be careful with the next exceptions that I write.